api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -333,7 +333,8 @@ message ExternalProcessor {
   // Decorator to introduce custom logic that runs after a message received from
   // the External Processor is processed.
   // [#extension-category: envoy.http.ext_proc.response_processors]
-  config.core.v3.TypedExtensionConfig on_processing_response = 23;
+  config.core.v3.TypedExtensionConfig on_processing_response = 23
+      [(xds.annotations.v3.field_status).work_in_progress = true];
```

Comment:
adding `work_in_progress` denotes that this field could be susceptible to breaking changes - is it okay to add this for use-cases that are already relying on on_processing_response?

Reply:
I think this feature is WIP, and does not work now.  It's better to mark it as WIP so folks won't accidentally try it.  We can remove it later on.

Reply:
This was added in https://github.com/envoyproxy/envoy/pull/37916 less than a month ago. In hindsight, the annotation should have been added in that PR. I don't think anyone else is using this extension point.

Reply:
@adisuissa WDYT? I added this now based on @yanjunxiang-google's comment.

Reply:
If the intention is to clarify that this is still under test, I suggest adding a note (see [here](https://github.com/envoyproxy/envoy/blob/53941d96922ae2bee829863929fc127921e167b3/api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto#L155-L156)).

Reply:
Added a note, PTAL

Reply:
I guess the WIP is not needed, but I don't think it matters much.
/lgtm api